force-app/main/default/staticresources/Figure.resource-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Alle de statisk ressursene virker ganske generiske, er dette noe du har laget for lesehjelp? Da bør de mer spesifikke navn, hvis det er noe generisk som finnes fra før av må de kanskje ligge i crm-community-base.

Reply:
Hentet de fra aksel for base hadde ikke noen fine. 

Kan du gi eksempel filnavn?
PilPointingToTheRightAndIsFilled.svg?

Reply:
Poeng var bare om dette er noen flere kommer til å kunne tenkte seg å bruke, så kunne man kanskje med fordelt ha lagt alle ikonene fra Aksel i crm-communuty-base.

Reply:
Er stor sannsynlighet for at det kanskje finnes noen med samme navn allerede.

Reply:
Så enten flytt de til en felles pakke, eller prefix dem med HOT_

Reply:
Kunne ikke endre størrelsen på ikonene fra de komponentene. Så har lagt til HOT_